css/navbar.css

```diff
@@ -0,0 +1,109 @@
+.navbar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  z-index: 2;
+  background-image: linear-gradient(
+    180deg,
+    #24211e 0%,
+    #5c544d8c 50%,
+    rgba(37, 33, 30, 0) 100%
+  );
+  transition: var(--transition-one);
+}
+
+.navbar * {
+  color: var(--clr-grey-10);
+}
+
+.navbar:hover {
+  background: var(--primary-color);
+}
+
+.nav {
+  padding: 0.625rem 1.5rem;
+  display: flex;
+}
+
+.information {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  margin-right: auto;
+  font-size: 12px;
+  z-index: 1;
+}
+
+.brand-logo {
+  position: absolute;
```

Comment:
why position the brand logo to absolute? I personally think it looks better without it.

Reply:
<img width="1426" alt="image" src="https://user-images.githubusercontent.com/38975782/153049443-9dcedd9e-7c76-4de4-8162-2942be374e32.png">

I think it is centered perfectly without it.

Reply:
did you try on the `anchor` link?
```margin-left: auto;
margin-right: auto;
```

Reply:
I'll look into it now, thank you for pointing it out @venkateshwarans.

Reply:
Hey @venkateshwarans, I have made the changes now, thank you for reviewing my code and UI.